frontend/views/containers/navigation/Profile.vue

```diff
@@ -98,7 +98,7 @@ export default {
 
 button {
   .icon-cog {
-    font-size: 0.9rem;
+    font-size: $size_4;
```

Comment:
These tiny adjustments break the UI. You'll notice that now the icon cog is not centered and when animated, it doesn't stay in the center either.